### ISSUE TITLE
Angular 2 version is old, please update code to latest

### ISSUE THREAD

**Issue by Cyberdada** [gdi2290/ng-vegas-angular2-d3#2]
This is really interesting code, however several versions of Angular 2 have been released after the code was written. please upgrade to latest version. :)

**Comment by PatrickJS** [comment]
yup, this wouldn't be too hard to update :+1: I'll try to get to it this weekend

**Comment by mgemard** [comment]
:+1:

**Comment by epyx25** [comment]
@gdi2290 Any news on this?

**Comment by anshulee** [comment]
We have put up a d3 Angular2 RC-1 version of this example at https://github.com/anshulee/angular2d3. @gdi2290 would love to get your views and send you a PR :-)
